AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -65,12 +68,103 @@ class ServiceLocatorTests : XCTestCase {
             XCTAssertEqual(rStringService, testServiceString)
             
         } catch let error {
-            XCTAssert(false, "💥: Register failed with error: \(error)")
+            XCTFail("💥: Register failed with error: \(error)")
         }
     }
-    
-    func testLazyInitRegisterUsingTypeInference() {
+
+    // MARK: Failure
+
+    func testRegister_UsingTypeInference_ShouldFailWithDuplicatedService() {
+
+        let testService = "Test Register Service"
+
+        do {
+
+            // Register the first service
+            try serviceLocator.register(service: testService)
+
+            // Register the same service, should fail with duplicateService
+            try serviceLocator.register(service: testService)
+
+        } catch ServiceLocatorError.duplicateService(_) {
+            print("✅"); return
```

Comment:
Here we could test the returned error itself and check if the associated value (the conflicting service name) matches `serviceName`.

On another point: do we really need to print something here? The test will pass if we simply leave this `catch` block empty 😉 

On a more 🚓 mode: dat `;` and think we don't need the `return` 😜 

Finally, we have a lot of these cases on the test suite, so the same comment applies to all